.github/ISSUE_TEMPLATE.yml

```diff
@@ -0,0 +1,89 @@
+name: Bug Report
+description: Create a bug report
+labels: ["defect"]
+body:
+  - type: textarea
+    attributes:
+      label: Description
+      description: "Please provide a description and way to reproduce the problem."
+      placeholder: |
+        Please use the Yii Chat channel or IRC channel for questions.
+        
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Yii Version
+      description: Which version of Yii are you using?
+      placeholder: "2.0.49 (latest)"
+      multiple: false
+      options:
+      - 2.0.49 (latest)
```

Comment:
We'll need to maintain that. I'd just ask to confirm that latest Yii version is used.

Reply:
Here we should to ask package version.

Reply:
As an example for Yii, I had thought about having a file "versions-yii.txt" with the list of Yii versions that are still supported to be loaded here. This file would be in a common repository, for example, "https://github.com/yiisoft/docs". Just like a "version-yii.txt" file with just the current version.

As an example for the package itself, I thought about having a file in the local repository like "versions.txt" with the list of versions of this package to be loaded here, with the versions that are still supported.

On the other hand.... a simple input field to write the version used is simpler.

Reply:
Since there are multiple packages, I'd just leave the field as freeform text without trying to make a list at all.

Reply:
> Since there are multiple packages, I'd just leave the field as freeform text without trying to make a list at all.

Sometimes what's easiest is what works best

Reply:
done